### PR TITLE
syncer: improve coverage for syncer/ddl.go to 85.1%

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
part of #998 

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: syncer/ddl_test.go [hunk 1]
```diff
@@ -358,3 +370,156 @@ func (s *testSyncerSuite) TestResolveGeneratedColumnSQL(c *C) {
...
+	}
+	syncer.onlineDDL = mock
+
+	_, _, _, _, err := syncer.sgk.AddGroup(targetDB, targetTbl, []string{source1}, nil, true)
+	c.Assert(err, IsNil)
+	_, _, _, _, err = syncer.sgk.AddGroup(targetDB, targetTbl, []string{source2}, nil, true)
+	c.Assert(err, IsNil)
+
+	c.Assert(syncer.clearOnlineDDL(tctx, targetDB, targetTbl), IsNil)
+	c.Assert(mock.toFinish, HasLen, 0)
```
Comment (csuzhangxc) [inline syncer/ddl_test.go:524]: It will always `HasLen` `0` because we never store anything to it?

**Review by csuzhangxc — COMMENTED** [review]
rest LGTM

File: syncer/ddl_test.go (hunk 1 above)
Comment (lance6716) [reply syncer/ddl_test.go:524]: it's in line 514, "finish" will remove one table from this member.

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**